DeAceroDLabs/marketplace-frontend
# [DMP-727] Added tooltip to "cambiar direccion".

## What is done

[[DMP-727] Added tooltip to "cambiar direccion".](https://deacero.atlassian.net/browse/DMP-727)

## How is done

- Used the same component as footer support.


## How to test

1. Run the project via `npm start`.
2. Add a product to shopping cart.
3. Go to payment click on "¿Quieres cambiar dirección?":
![image](https://user-images.githubusercontent.com/81311646/206049125-4228dfe4-5329-43ba-a425-a88cd15fabc4.png)

5. You should see a popup.
